### DISCUSSION TITLE
IS THER ANY WAY TO HOOK THIS UP TO LANGCHAIN OR SMOL DEV   would love a way to hook this up to SMOL DEV and Langchain in order to have Agents work on projects that are outlined in PLANE and have th SMOL DEV code the project recursively

### DISCUSSION THREAD

**Discussion by jmanhype** [makeplane/plane#1201] [Ideas]
IS THER ANY WAY TO HOOK THIS UP TO LANGCHAIN OR SMOL DEV 

would love a way to hook this up to SMOL DEV and Langchain in order to have Agents work on projects that are outlined in PLANE and have th SMOL DEV code the project recursively

**Comment by pablohashescobar** [comment]
that directly connects with Langchain. However, I want to assure you that it's already on our roadmap, and we have plans to integrate with Langchain in the near future. Stay tuned for updates as we work towards implementing this feature.
